src/components/Layout.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {View, ViewStyle} from 'react-native'
+import {StyleProp} from 'react-native'
+import {useSafeAreaInsets} from 'react-native-safe-area-context'
+
+import {atoms as a} from '#/alf'
+
+// Every screen should have a Layout component wrapping it.
+// This component provides a default padding for the top of the screen.
+// This allows certain screens to avoid
```

Comment:
I think there might be something missing in this sentence? 🤔

Reply:
lol yeah thanks

Reply:
Also: this is one of those PRs where Hide whitespace **really** comes in handy 😅